src/applications/vaos/tests/new-appointment/components/CommunityCareProviderSelectionPage/index.unit.spec.jsx

```diff
@@ -121,6 +123,35 @@ describe('VAOS <CommunityCareProviderSelectionPage>', () => {
 
     expect((await screen.findAllByRole('alert')).length).to.equal(2);
     expect(screen.history.push.called).to.be.false;
+
+    // Continue with filling in required fields without provider
+    userEvent.click(await screen.getByRole('radio', { name: /Bozeman, MT/i }));
+    const languageSelect = screen.getByLabelText(
+      /do you prefer that your community care provider speak a certain language?/i,
+    );
+
+    fireEvent.change(languageSelect, { target: { value: 'english' } });
+    userEvent.click(screen.getByText(/Continue/i));
+    expect(global.window.dataLayer[2].event).to.equal(
+      `${GA_PREFIX}-continue-without-provider`,
+    );
+    expect(screen.history.push.called).to.be.true;
+
+    // Continue with filling in required fields with provider
+    userEvent.click(await screen.findByText(/Choose a provider/i));
+    userEvent.click(await screen.findByText(/AJADI, ADEDIWURA/i));
+    userEvent.click(
+      await screen.getByRole('button', { name: /choose provider/i }),
+    );
+    expect(await screen.baseElement).to.contain.text(
+      'AJADI, ADEDIWURA700 CONSTITUTION AVE NEWASHINGTON, DC 20002-65999349.3 miles',
+    );
+
+    userEvent.click(screen.getByText(/Continue/i));
+    expect(global.window.dataLayer[5].event).to.equal(
```

Comment:
Because this is a global value, it persists between tests, which means that other tests might put stuff in here that messes up the order. I would do something like a `.some` call on `dataLayer` to find that there's some event somewhere in here that matches what you're looking for.